projects/email-editor/src/lib/email-editor.component.ts

```diff
@@ -8,7 +8,6 @@ import {
 } from '@angular/core';
 import { loadScript } from './loadScript';
 import pkg from './source.json';
-import { Editor, UnlayerOptions, ToolsConfig, JSONTemplate } from '../types';
```

Comment:
We can keep the `types` file with all those types there, but copy pasted from `unlayer-types` for now